es6-modules/src/components/todo-app.js

```diff
@@ -0,0 +1,59 @@
+// const { createElement, noop } = require('../lib');
+import { createElement, noop } from '../lib/index';
```

Comment:
When you are importing from an `index` file you don't need to specify the filename, for example:

```JAVASCRIPT
import { createElement, noop } from '../lib';
```